public/utils/report_metric.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { v4 as uuidv4 } from 'uuid';
+import { UsageCollectionSetup } from '../../../../src/plugins/usage_collection/public';
+
+export const reportMetric = (
+  usageCollection?: UsageCollectionSetup,
+  metricAppName: string = 'app',
+  metric: string = 'click'
+) => {
+  if (usageCollection) {
+    usageCollection.reportUiStats(
+      metricAppName,
+      usageCollection.METRIC_TYPE.CLICK,
```

Comment:
I think metric type is configurable in some cases. You can make it parameter with default value as well.

Reply:
ok, I will add an argument for it